src/_base/helm/app/templates/application/webapp/ingress.yaml

```diff
@@ -1,27 +1,29 @@
 {{ if and (index .Values.services .Values.ingress.target_service "enabled") (eq .Values.ingress.type "standard") }}
+{{- with .Values.services.nginx -}}
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-{{- if .Values.services.nginx.ingress.annotations }}
+{{- if .ingress.annotations }}
   annotations:
-  {{- range $key, $value := .Values.services.nginx.ingress.annotations }}
```

Comment:
what do you think to moving .Values.services.nginx.ingress.annotations to .Values.ingress.annotations for the 1.0.0 release?

Reply:
I guess there might be annotations for the istio ingress too, though that can be switched at the values.yaml.twig level

Reply:
istio virtualservice only has labels atm

Reply:
How does https://github.com/inviqa/harness-base-php/pull/538 look?

Reply:
It's now part of this PR :)